Documentation/ApiOverview/Mail/Index.rst

```diff
@@ -26,55 +15,58 @@ for creating email messages.
 TYPO3’s backend functionality already ships with a default layout for templated emails,
 which can be tested out in TYPO3’s install tool test email functionality.
 
-.. contents:: Table of Contents
-   :depth: 1
-   :local:
+..  contents:: Table of Contents
+    :depth: 1
+    :local:
 
 
-.. index::
-   pair: Mail; Configuration
-   TYPO3_CONF_VARS; MAIL
-.. _mail-configuration:
+..  index::
+    pair: Mail; Configuration
+    TYPO3_CONF_VARS; MAIL
+..  _mail-configuration:
 
 Configuration
 =============
 
-Several settings are available in the "Configure Installation-Wide Options"
-:php:`$GLOBALS['TYPO3_CONF_VARS']['MAIL']`.
+Several settings are available via :guilabel:`Admin Tools > Settings > Configure
+Installation-Wide Options > Mail` which are stored into
+:php:`$GLOBALS['TYPO3_CONF_VARS']['MAIL']`. See :ref:`typo3ConfVars_mail` for
+an overview of all settings.
 
 
-.. _mail-configuration-format:
+..  _mail-configuration-format:
 
 Format
 ------
 
-:php:`$GLOBALS['TYPO3_CONF_VARS']['MAIL']['format']` can be `both`,
-`plain` or `html`. This option can be overridden by Extension authors
-in their use cases.
+:php:`$GLOBALS['TYPO3_CONF_VARS']['MAIL']['format']` can be `both`, `plain` or
+`html`. This option can be overridden by extension authors in their use cases.
```

Comment:
Should it really be overridden in an Extension and not rather in the LocalConfiguration / Additional Configuration by administrators?

Reply:
Good catch.